main/src/ca/uwaterloo/flix/language/errors/NameError.scala

```diff
@@ -69,25 +70,29 @@ object NameError {
     * @param loc2 the location of the second definition.
     */
   case class DuplicateDefOrSig(name: String, loc1: SourceLocation, loc2: SourceLocation) extends NameError {
-    def summary: String = s"Duplicate definition."
+    def summary: String = s"Duplicate definition of '$name'."
 
     def message(formatter: Formatter): String = {
       import formatter._
       s"""${line(kind, source.format)}
-         |>> Duplicate definition '${red(name)}'.
+         |>> Duplicate definition of '${red(name)}'.
          |
-         |${code(loc1, "the first occurrence was here.")}
+         |${code(loc1, "the first definition was here.")}
          |
-         |${code(loc2, "the second occurrence was here.")}
+         |${code(loc2, "the second definition was here.")}
          |""".stripMargin
     }
 
-    def loc: SourceLocation = loc1
-
     def explain(formatter: Formatter): Option[String] = Some({
-      import formatter._
-      s"${underline("Tip:")} Remove or rename one of the occurrences."
+      """Note: Flix does not support function overloading, i.e. you cannot define two
```

Comment:
Maybe the "Note:" should be removed as it's not present in any other explanation?